integration-boilerplate/src/backend/backend.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { assert } from 'chai';
+import * as express from 'express';
+import { createTestkit, IntegrationsTestkit } from 'wix-answers-integrations-testkit';
+import { initAnswersApi } from './backend';
+
+const testConfig = {
+	answersApiSecret: 'mXYjQ3DPRNK4tvrq-LFM2d5ZO5_M03yzvtvnxqrtsCI',
+	apiPort: 3004,
+	integrationId: '1234',
+	baseUrl: 'http://localhost',
+	ecryptKey: 'testssEXAMPLE',
+	routes: {
+		path: 'answers/api',
+		register: 'register',
+		unregister: 'unregister',
+		settings: 'settings',
+		script: 'script.js',
+		view: 'view',
+	},
+	mongo: {
+		mongoUrl: '',
+		initDataDB: '',
+		settingsDB: '',
+	}
+};
+
+const baseUrl = `http://localhost:${testConfig.apiPort}/${testConfig.routes.path}`;
+
+const initConfig = {
+	id: testConfig.integrationId,
+	secret: testConfig.answersApiSecret,
+	scriptUrl: `${baseUrl}/${testConfig.routes.script}`,
+	settingsUrl: `${baseUrl}/${testConfig.routes.settings}`,
+	registerUrl: `${baseUrl}/${testConfig.routes.register}`,
+	unregisterUrl: `${baseUrl}/${testConfig.routes.unregister}`
+};
+
+const payload = {
+	keyId: 'myKeyId',
+	secret: 'mySecret',
+	host: 'test.wixamswers.com',
+	tenantId: '123333221'
+};
+
+class MongoWrapper {
+	initDB = {};
+	settingsDB = {};
+	registerTenant = async (data): Promise<any> => {
+		if (!this.initDB[data.tenantId]) {
+			this.initDB[data.tenantId] = data;
+			return data.tenantId;
+		}
+		return;
+	}
+	dangerouslyGetAppKeys = async (tenantId: string) => this.initDB[tenantId] && delete this.initDB[tenantId];
```

Comment:
why `dangerouslyGetAppKeys`? do we need that name?

Reply:
because I loved the 'dangerously' but maybe shorter name will be better ^^